ci: Use reusable Docs CI workflow and add Vale linting with PR suggestions (#6139)
grafana/alloy: docs/sources/set-up/supported-platforms.md

# Supported platforms

> The following operating systems and hardware architectures are supported by Alloy's collector pipeline.

Reviewer: ```suggestion
The following operating systems and hardware architectures are supported by the {{< param "PRODUCT_NAME" >}} collector pipeline.
```

Author: Haha, sorry for giving you docs to review, mostly only made the change to demonstrate the linting PR comment. Happy to drop it. Or make it active voice if we want 😂 :

```suggestion
The {{< param "PRODUCT_NAME" >}} collector pipeline supports the following operating systems and hardware architectures.
```

Reviewer: Either way is good :-)
